src/ClickatellClient.php

```diff
@@ -3,7 +3,8 @@
 namespace NotificationChannels\Clickatell;
 
 use stdClass;
-use Clickatell\Api\ClickatellHttp;
+use Clickatell\ClickatellException;
+use Clickatell\Rest;
```

Comment:
IMO we should alias this import to something a bit more logical

Reply:
Agreed. I'm thinking ClickatellRest?

Reply:
Yep, sounds good to me.[](url)